v2.6/getting-started/kubernetes/upgrade/convert.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Converting policies
+redirect_from: latest/getting-started/kubernetes/upgrade/convert
+---
+
+If you are upgrading from the following, you must convert your policies
```

Comment:
We need to make it really clear that this is a one-time operation that only needs to happen if you're upgrading from a really old version of Calico.

I almost hesitate to include the instructions in the v2.6 documentation because we typically don't support direct upgrades from versions that are so old.